core/templates/dev/head/pages/exploration_player/AnswerClassificationService.js

```diff
@@ -107,31 +110,31 @@ oppia.factory('AnswerClassificationService', [
        * @param {function} interactionRulesService - The service which contains
        *   the explicit rules of that interaction.
        *
-       * @return {promise} A promise for an AnswerClassificationResult domain
-       *   object.
+       * @return {AnswerClassificationResult} The resulting
```

Comment:
Considering new changes, this should be 'AnswerClassificationResultObjectFactory'. Ditto below.